code/datums/wounds/burns.dm

```diff
@@ -52,18 +52,29 @@
 	if(limb.current_gauze)
 		limb.seep_gauze(WOUND_BURN_SANITIZATION_RATE)
 
-	if(flesh_healing > 0)
+	if(flesh_healing > 0) // good bandages multiply the length of flesh healing
 		var/bandage_factor = (limb.current_gauze ? limb.current_gauze.splint_factor : 1)
 		flesh_damage = max(0, flesh_damage - 1)
-		flesh_healing = max(0, flesh_healing - bandage_factor) // good bandages multiply the length of flesh healing
+		flesh_healing = max(0, flesh_healing - bandage_factor)
+
+	if(infestation <= WOUND_INFECTION_MODERATE && limb.burn_dam < 10) // if we have little/no infection, the limb doesn't have much burn damage, and our nutrition is good, heal some flesh
+		switch(victim.nutrition)
+			if(NUTRITION_LEVEL_FED to NUTRITION_LEVEL_WELL_FED)
+				flesh_healing += 0.6
```

Comment:
Move 0.6 out to its own define if you're going to be using it multiple times.

Reply:
Also, just to be clear, this code means that if you've got a "good" amount of nutrition your burns heal, but if you're stuffing your face with ice cream, they are less likely to heal?

Reply:
I just narrowed it down to having at least fed and all being the same rate